lib/LedgerSMB/Upgrade_Tests.pm

```diff
@@ -546,6 +619,104 @@ number of "AR_*", "AP_*" and/or "IC_*" links concatenated by colons (:).'),
     max_version => '3.0'
     );
 
+push @tests,__PACKAGE__->new(
+    test_query => q{select *, (select array_agg(x in (
+                            SELECT value FROM pg_temp.account_link
+                           )) from unnest(string_to_array(c.link,':')) x)
+                        from chart c
+                       where ((select array_agg(x in (
+                            SELECT value FROM pg_temp.account_link
+                           )) from unnest(string_to_array(c.link,':')) x) @> ARRAY[0]::bool[]
+                           or c.link is null
+                           or c.link = '')
+                         and charttype = 'A'
+                    order by accno;
+                    },
+    display_name => marktext('No unlinked account'),
+    name => 'no_unlinked_account',
```

Comment:
I think this check is incorrect: accounts are allowed to be unlinked. In that case they only have "manual journal" value, but I have many accounts like that myself... Nothing to be worried about.

Reply:
Moved in another PR